.textlintrc

```diff
@@ -92,7 +97,9 @@
         ["can[- ]not", "cannot$1"],
 
         // Multiple words
+        ["back-?end(s?)", "back end$1"],
```

Comment:
I never know which version to use

Reply:
I just picked having them as multiple words. I'm not really sure there is a right answer - but we want consistency anyways.